src/amo/components/UserProfileEditPicture/index.js

```diff
@@ -67,6 +67,8 @@ export class UserProfileEditPictureBase extends React.Component<InternalProps> {
       },
     );
 
+    const ConfirmButtonClassName = 'UserProfileEditPicture-delete-button';
```

Comment:
nitpick: variable names should always be camel case starting with a lower case letter unless they refer to a class.